vm/src/dictdatatype.rs

```diff
@@ -271,6 +273,7 @@ enum LookupResult {
     Existing(EntryIndex), // Existing record, index into entries
 }
```

Comment:
The string passed to `flame` is a prefix, so this would appear in the output as `collection_hash::collection_hash`. I'd recommend changing it to `dictdatatype` or removing the prefix altogether.